service/javadsl/api/src/main/java/com/lightbend/lagom/javadsl/api/transport/TransportErrorCode.java

```diff
@@ -86,7 +86,7 @@
      * A generic error used to indicate that the end sending the error message because it encountered an unexpected
      * condition.
      */
-    public static final TransportErrorCode UnexpectedCondition = new TransportErrorCode(500, 1011, "Unexpected Condition");
+    public static final TransportErrorCode UnexpectedCondition = new TransportErrorCode(500, 1011, "Unexpected Condition/Internal Server Error");
 
```

Comment:
`UnexpectedCondition` and `PolicyViolation` use error codes that have different names depending on the protocol. Because we have alias variables (one per name) the description is better if it includes both possible descriptions (same approach was used in other places in the same file).

Reply:
@ignasi35 you've changed the `TransportErrorCode#description` here (in Java) but not in the Scala side, is it normal ? See: https://github.com/lagom/lagom/blob/master/service/scaladsl/api/src/main/scala/com/lightbend/lagom/scaladsl/api/transport/Exceptions.scala#L102

Reply:
Same question for `PolicyViolation`: https://github.com/lagom/lagom/blob/master/service/scaladsl/api/src/main/scala/com/lightbend/lagom/scaladsl/api/transport/Exceptions.scala#L77

Reply:
That's an oversight. Thanks for pointing this out!